pkg/action/executor/rescheduler.go

```diff
@@ -214,12 +214,12 @@ func (r *ReScheduler) reSchedule(action *turboaction.TurboAction) (*turboaction.
 	}
 
 	preScheduler, err := f(r.kubeClient, namespace, parentName, "", DefaultNoneExistSchedulerName)
+	defer f(r.kubeClient, namespace, parentName, DefaultNoneExistSchedulerName, preScheduler)
```

Comment:
OK, got it. I think a better way to write this is 
```go
preScheduler := getScheduler()
defer resetSchedler(preScheduler)
setScheduler(DefaultNoneExistSchedulerName)
```

But this way also works and I am fine with it.